prog/postgres/postgres_server_nexus.rb

```diff
@@ -166,10 +166,44 @@ def before_run
 CONFIG
     vm.sshable.cmd("sudo -u prometheus tee /home/prometheus/web-config.yml > /dev/null", stdin: web_config)
 
-    vm.sshable.cmd("sudo systemctl enable --now postgres_exporter")
-    vm.sshable.cmd("sudo systemctl enable --now node_exporter")
-    vm.sshable.cmd("sudo systemctl enable --now prometheus")
-    hop_configure
+    metric_destinations = postgres_server.resource.metric_destinations.map {
+      <<METRIC_DESTINATION
+- url: '#{_1.url}'
+  basic_auth:
+    username: '#{_1.username}'
+    password: '#{_1.password}'
+METRIC_DESTINATION
+    }.prepend("remote_write:").join("\n")
+
+    prometheus_config = <<CONFIG
+global:
+  scrape_interval: 15s
+  external_labels:
+    resource: #{postgres_server.resource.ubid}
+    role: #{postgres_server.primary? ? "primary" : "standby"}
```

Comment:
If there are multiple standbys, should we differentiate their metrics?

Reply:
I didn't want to expose postgres_server's ubid, because so far, user's all interactions were against the resource, but it seems we have to for this case. I added an instance field to distinguish different standbys.